### PR TITLE
Create animal table 

### PR DESCRIPTION
**Pull request by EyasuTeshome**
## Animal Table

I have attached the screenshot of all the queries below. and I have included all the `SQL` I used in `data.sql`, `queries.sql`, and `schema.sql`.

## Screenshoots 

- Find all animals whose name ends in "mon".
Screen Shot 2022-09-20 at 12 08 12

- List the name of all animals born between 2016 and 2019.
Screen Shot 2022-09-20 at 12 12 55

- List the name of all animals that are neutered and have less than 3 escape attempts.
Screen Shot 2022-09-20 at 12 47 00

- List the date of birth of all animals named either "Agumon" or "Pikachu".
Screen Shot 2022-09-20 at 12 19 48

- List name and escape attempts of animals that weigh more than 10.5kg
Screen Shot 2022-09-20 at 12 22 37

- Find all animals that are neutered.
Screen Shot 2022-09-20 at 12 23 56

- Find all animals not named Gabumon.
Screen Shot 2022-09-20 at 12 25 10

- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
Screen Shot 2022-09-20 at 12 50 17

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE %mon;
+SELECT * FROM animals WHERE date_of_birth BETWEEN CAST('2016-01-01' AS DATE) AND CAST('2019-12-30' AS DATE);
+SELECT * FROM animals WHERE neutered = false AND escape_attempts < 3;
```
Comment (sja-thedude) [inline queries.sql]: - [ ] Kindly make sure that neutered = true as the query needs to display all animals that are neutered.
- [ ] Make sure to add the updated screenshot for this query.

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE %mon;
+SELECT * FROM animals WHERE date_of_birth BETWEEN CAST('2016-01-01' AS DATE) AND CAST('2019-12-30' AS DATE);
```
Comment (sja-thedude) [inline queries.sql]: - [ ] Kindly make sure that the end date is 31 Dec. Currently is 30 Dec.
- [ ] Make sure to add the updated screenshot for this query.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @EyasuTeshome, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct gitflow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (Bondok6) [reply queries.sql]: - [ ] As the previous reviewer mentioned, Kindly change the end date to `2019-12-31` instead of `2019-12-30` 💯

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @EyasuTeshome 🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
- [ ] _Check the comments under the review._ ⬇

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @EyasuTeshome ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**